### PR TITLE
Scroll up player list when searching

### PR DESCRIPTION
**Pull request by turbedi**
Fix issue #153

### CODE REVIEW

**Comment by coyotte508** [comment]
you should check it doesn't start with # (channel) and isn't empty

**Comment by turbedi** [comment]
Updated.

File: app/assets/javascript/playerlistui.js [hunk 1]
```diff
@@ -181,8 +181,13 @@ $(function() {
     webclientUI.players.count = $("#playercount");
 
     $("#player-filter").on("input", function() {
+        if (/^\s*$/.test($("#player-filter").val())) { /* Don't search when the string is blank */
+            return;
```
Comment (coyotte508) [inline app/assets/javascript/playerlistui.js]: actually no,  the setFilter should still be called even if to just reset the filter to blank.

**Comment by coyotte508** [comment]
``` js
var s = $("#player-filter").val();
if (s.length > 0 && s[0] != '#') {
   //scroll up
}
```

**Comment by turbedi** [comment]
Updated again

**Pull request merged**